lib/generator.js

```diff
@@ -121,7 +121,7 @@ class Generator {
           await this.launchHook('generate:after');
           return;
         } else if (this.output === 'string') {
-          return this.renderString(asyncapiDocument, await readFile(entrypointPath, { encoding: 'utf8' }), entrypointPath);
+          return this.renderString(asyncapiDocument, await readFile(entrypointPath, { encoding: 'utf8' }));
```

Comment:
It's important we keep `entrypointPath` there, remember? https://github.com/asyncapi/generator/pull/146

Reply:
Yep absolutely right, no idea why this was overwritten 🤔